e2e-tests/submit-to-trade-finance-manager/cypress/integration/journeys/portal/submit-AIN-deal-creates-default-credit-rating-in-tfm/submit-AIN-deal-creates-default-credit-rating-in-tfm.spec.js

```diff
@@ -63,6 +63,8 @@ context('Portal to TFM deal submission', () => {
     // expect to land on the /dashboard page with a success message
     cy.url().should('include', '/dashboard');
 
+    cy.wait(5000); // wait for TFM to do it's thing
```

Comment:
We don't need these wait() cmds now. Cypress waits for the page to load based on the pageLoadTimeout setting in cypress.json. Its recently been set to 4mins

Reply:
thanks, removed - just added them to try and help the tests pass in github